doc/api/crypto.md

```diff
@@ -1388,7 +1388,7 @@ See the reference for other recommendations and details.
 
```

Comment:
Should I amend the title, or submit an additional PR for the trailing whitespace removal?